lib/schemaUtils.js

```diff
@@ -179,7 +179,8 @@ function safeSchemaFaker (oldSchema, resolveTo, resolveFor, parameterSourceOptio
   if (resolveTo === 'schema') {
     key = 'resolveToSchema ' + key;
     schemaFaker.option({
-      useExamplesValue: false
+      useExamplesValue: false,
```

Comment:
This change has been done to make sure correct JSF options are set. There was issue where, due to new introduced interfaces, the options that were once set were persisting for next unit test-cases.

With this we are always making sure such cases doesn't happen. For isolated execution, this code doesn't affect anything as we're setting same option with sam value globally.